app/views/mentee_profiles/show.html.slim

```diff
@@ -31,4 +35,9 @@
     p.top-space-20
       strong Mentor:
       =< link_to @mentor.full_name, mentor_profile_path(@mentor)
+      - if current_user.role == "organizer" && @mentor.is_missing
+         .pull-right
+           | Missing since 
+           = @mentor.missing_since.strftime("%B %d, %Y")
```

Comment:
Right now, I have made the functionality that when 
mentor presses abandoned project, corresponding to it is shown on mentee profiles. 
Should i also show on mentor profile in front of mentee name ? 
Also how to display text in red :P